templates/default/keystone-v2.cfg.xml.erb

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<keystone-v2 xmlns="http://docs.openrepose.org/repose/keystone-v2/v1.0">
+    <identity-service
+        username="<%= @username %>"
+        password="<%= @password %>"
+        uri="<%= @uri %>"
+        connection-pool-id="<%= @pool %>"
+        set-roles-in-header="<%= @roles_in_header %>"
+        set-groups-in-header="<%= @groups_in_header %>"
+        set-catalog-in-header="<%= @catalog_in_header %>"
+        />    
+    <% if @delegating_quality %>
+    <delegating quality="<%= @delegating_quality %>" /> 
+    <% end %>
+    
+    <% if @white_list %>
+    <white-list>
+	<% @white_list.each do | uri_regex | %>
+        <uri-pattern uri-regex="<%= uri_regex %>" />
+	<% end %>
+    </white-list>
+    <% end %>
+
+    <% if @cache %>
+     <cache>
+        <timeouts variability="<%= @cache['timeout_variability'] %>">
+            <token><%= @cache['token_timeout'] %></token>
+	    <group><%= @cache['group_timeout'] %></group>
+	    <endpoints><%= @cache['endpoints_timeout'] %></endpoints>
+        </timeouts>
+          <% if @cache['atom_feeds']
+	  @cache['atom_feeds'].each do | atom_feed | %>
+        <atom-feed id="<%= atom_feed %>"/>
+          <% end
+	    end %>
+    </cache>
+    <% end %>
+
+    <% if @tenant_handling %>
+    <tenant-handling send-all-tenant-ids="<%= @tenant_handling['send_all_tenant_ids'] %>">
+        <% if @tenant_handling['validate_tenant'] %>
+        <validate-tenant strip-token-tenant-prefixes="<%= @@tenant_handling['validate_tenant']['strip_token_tenant_prefixes'] %>">
```

Comment:
I checked and the whole tenant_handling is nil.  All you have to do is not define it in your hash and everything should work.  Please let me know if I'm wrong but it tested out for me.